dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.channelfw-1.0.feature

```diff
@@ -4,6 +4,7 @@ WLP-DisableAllFeatures-OnConflict: false
 IBM-API-Package: com.ibm.websphere.endpoint; type="ibm-api"
 IBM-Process-Types: server, \
  client
+-features=com.ibm.ws.wsbytebuffer-1.0
```

Comment:
I think we should do this via a bundle unless we use some of the coreq/auto feature function that needs a buffer feature in order to easily test different implementations.

Reply:
Gone.